README.md

```diff
@@ -79,7 +82,7 @@ No PetriNet is generated / there is an error while generating the net
 > Check if you correctly installed GraphViz (On Windows you need to put the bin folder inside the GraphViz folder into the PATH environment variable. On Ubuntu it should be fine if you install GraphViz via apt-get)
 
 ## License
-PDFL is licensed under the MIT License. See [LICENSE](LICENSE) for details on the licensing terms.
+PDFL is licensed under the MIT License. See [LICENSE](https://github.com/iml130/pfdl/blob/main/LICENSE) for details on the licensing terms.
```

Comment:
Do you need an absolute path here?

Reply:
At the moment, the README content is used as the long description for PyPi. Here, the absolute path is needed because the relative paths are not resolved.